social/exceptions.py

```diff
@@ -1,5 +1,39 @@
-from social_core.exceptions import SocialAuthBaseException, WrongBackend, \
-    MissingBackend, NotAllowedToDisconnect, AuthException, AuthFailed, \
-    AuthCanceled, AuthUnknownError, AuthTokenError, AuthMissingParameter, \
-    AuthStateMissing, AuthStateForbidden, AuthAlreadyAssociated, \
-    AuthTokenRevoked, AuthForbidden, AuthUnreachableProvider, InvalidEmail
+from social_core.exceptions import AuthAlreadyAssociated
+from social_core.exceptions import AuthCanceled as AuthCanceledBase
+from social_core.exceptions import AuthException
+from social_core.exceptions import AuthFailed as AuthFailedBase
+from social_core.exceptions import AuthForbidden
+from social_core.exceptions import AuthMissingParameter
+from social_core.exceptions import AuthStateForbidden
+from social_core.exceptions import AuthStateMissing
+from social_core.exceptions import AuthTokenError
+from social_core.exceptions import AuthTokenRevoked
+from social_core.exceptions import AuthUnknownError
+from social_core.exceptions import AuthUnreachableProvider
+from social_core.exceptions import InvalidEmail
+from social_core.exceptions import MissingBackend
+from social_core.exceptions import NotAllowedToDisconnect
+from social_core.exceptions import SocialAuthBaseException
+from social_core.exceptions import WrongBackend
+
+
+class AuthFailed(AuthFailedBase):
+    """Auth process failed for some reason."""
+    def __str__(self):
+        msg = super(AuthFailed, self).__str__()
+        if msg == 'access_denied':
+            return 'Authentication process was cancelled'
```

Comment:
I'm not sure in what situation this fail class would be called, but could we please change the message to `Connecting to your account has been cancelled`? Or maybe `Connecting to your account has failed`?